api/src/beacons/beacon.schema.ts

```diff
@@ -16,6 +16,9 @@ export class Beacon {
   @Prop({ required: true })
   description: string;
 
+  @Prop({ required: false })
+  tags: [string];
```

Comment:
Do we want to make tags an array of any strings, or only certain predetermined strings. If it's the latter, which I am more in favour of, we should make an enum with the possible tags and reference that here.